modules/acquia_cms_tour/src/Form/AcquiaSearchForm.php

```diff
@@ -96,54 +38,64 @@ protected function getEditableConfigNames() {
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
     $form['#tree'] = FALSE;
-    $module = 'acquia_search';
-    if ($this->module_handler->moduleExists($module)) {
+    $module = $this->module;
+    if ($this->isModuleEnabled()) {
       $module_path = $this->module_handler->getModule($module)->getPathname();
       $module_info = $this->infoParser->parse($module_path);
-      $form['acquia_search'] = [
-        '#type' => 'fieldset',
+      $configured = $this->getConfigurationState();
+      if ($configured) {
+        $form['check_icon'] = [
+          '#prefix' => '<span class= "dashboard-check-icon">',
+          '#suffix' => "</span>",
+        ];
+      }
+      $form[$module] = [
+        '#type' => 'details',
         '#title' => $module_info['name'],
-        '#description' => $module_info['description'],
-        '#open' => TRUE,
+        '#collapsible' => TRUE,
+        '#collapsed' => TRUE,
       ];
-      $form['acquia_search']['identifier'] = [
+      $form[$module]['identifier'] = [
         '#type' => 'textfield',
         '#required' => TRUE,
         '#title' => $this->t('Acquia Subscription identifier'),
-        '#default_value' => $this->state->get('acquia_search.identifier'),
-        '#description' => $this->t('Obtain this from the "Product Keys" section of the Acquia Cloud UI. Example: ABCD-12345'),
```

Comment:
@chandan-singh7929 Don't we need the description text for this field ?

Reply:
Done